  public void testExceptions() throws Exception {
    checkOneTerm(a, "καθεστώτα", "καθεστ");
    checkOneTerm(a, "καθεστώτος", "καθεστ");
    checkOneTerm(a, "καθεστώς", "καθεστ");
    checkOneTerm(a, "καθεστώτων", "καθεστ");

    checkOneTerm(a, "χουμε", "χουμ");
    checkOneTerm(a, "χουμ", "χουμ");

    checkOneTerm(a, "υποταγεσ", "υποταγ");
    checkOneTerm(a, "υποταγ", "υποταγ");

    checkOneTerm(a, "εμετε", "εμετ");
    checkOneTerm(a, "εμετ", "εμετ");

    checkOneTerm(a, "αρχοντασ", "αρχοντ");
    checkOneTerm(a, "αρχοντων", "αρχοντ");
  }

